internal/olm/operator/bundle/install.go

```diff
@@ -145,11 +124,11 @@ func (i *Install) setup(ctx context.Context) error {
 		if _, hasChannelMetadata := labels[registrybundle.ChannelsLabel]; hasChannelMetadata {
 			f.ChannelName = strings.Split(labels[registrybundle.ChannelsLabel], ",")[0]
 		} else {
-			f.ChannelName = defaultChannel
+			f.ChannelName = registry.DefaultChannel
 		}
 
 		// generate an fbc if an fbc specific label is found on the image or for a default index image.
-		content, err := f.generateFBCContent(ctx, i.BundleImage, i.IndexImageCatalogCreator.IndexImage)
+		content, err := generateFBCContent(f, ctx, i.BundleImage, i.IndexImageCatalogCreator.IndexImage)
```

Comment:
Any specific reason for passing `FBCContext` as an argument  instead of it being a struct receiver to the method?

Reply:
Since the `FBCContext` struct definition was moved to `index_image.go`, we can no longer define this function on that object in this file. So I changed it to be a function that takes `FBCContext` as an argument.

Reply:
@rashmigottipati is the above comment a valid explanation? If yes, I'll resolve this conversation. If not, happy to discuss it further.

Reply:
@VenkatRamaraju Yea, that makes sense. I'm good with the change.